additional_recipes/dynmsg/recipe.yaml

```diff
@@ -0,0 +1,79 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/prefix-dev/recipe-format/main/schema.json
+schema_version: 1
+
+package:
+  name: dynmsg
+  version: "1.0" # TODO Check this one
+
+source:
+  git: https://github.com/osrf/dynamic_message_introspection.git
+  rev: main
```

Comment:
I see that the upstream did not do any tag, but perhaps we can use a fixed checkout to avoid regressions?

Reply:
By the way, probably you are aware of this, but it seems that there are good news, someone is interested in mantaining the repo and actually do releases of it: https://discourse.ros.org/t/osrf-dynamic-message-introspection-maintainer-binary-release/42149  https://github.com/osrf/dynamic_message_introspection/issues/29 .